analysis/report_templates/default.html

```diff
@@ -96,6 +96,13 @@ <h5 class="center-align">By avg. rank</h5>
                     {{ experiment.rank_by_median_and_average_rank.round(2).to_frame().to_html() }}
                 </div>
             </div>
+            
+            <div class="row">
+                <div class="col s5 offset-s1">
+                    <h5 class="center-align">By avg. score on unique coverage</h5>
+                    {{ experiment.rank_by_unique_coverage_average_normalized_score.round(2).to_frame().to_html() }}
+                </div>
+            </div>
```

Comment:
Considering adding this to `analysis/report_templates/experimental.html` instead.

I don't think ranking by unique coverage can be considered a typical / usual way of ranking fuzzers. I wouldn't include it by default in the main report.

Reply:
+++++++1, still very early and we need to verify results and don't think it should be used for ranking.

Reply:
Got it!